scripts/requirements-extras.txt

```diff
@@ -16,4 +16,4 @@ lpc_checksum
 Pillow
 
 # can be used to sign a Zephyr application binary for consumption by a bootloader
-imgtool
+imgtool>=1.8.0a1
```

Comment:
Wait, we're setting the minimum version to an unreleased alpha?

What's going on here?